src/components/Loading/index.js

```diff
@@ -1,9 +1,43 @@
 import React from 'react';
+import CircleProgressBar from 'components/CircleProgressBar';
+import styles from './loading.scss';
 
-const Loading = ({ isLoading, children, onServer }) => (
-  onServer || (__CLIENT__ && isLoading) || !children
-    ? <div>Loading screen</div>
-    : children
-);
+const Loading = React.createClass({
+
+  getInitialState () {
+    return { percentage: 75 };
+  },
+
+  tick () {
+    this.setState({ percentage: this.state.percentage + 125 });
```

Comment:
I think you're right Redux is a bit overkill for this, unless we are measuring the actual progress of the API requests by bytes complete or so 😄 

I might have just gone for a CSS-only solution and I don't quite get the metrics of the initial percentage being `75` and adding `125` - can you clarify? But seems alright to me overall!